domains/daniel-mora.json

```
@@ -0,0 +1,12 @@
+{
+  "owner": {
+    "username": "dani_mc",
+    "email": "daniel_k310a@hotmail.com"
+  },
+  "record": {
+    "A":["162.241.62.192"],
+    "CNAME": {
+      "www": "daniel-mora.is-a.dev"
+    }
```

```suggestion
    "CNAME": {
      "www": "daniel-mora.is-a.dev"
    }
```
1. CNAME values must be strings, not dictionaries.
2. A CNAME cannot point towards itself.